### ISSUE TITLE
Add ability to make link components bold

### ISSUE THREAD

**Issue by kyuhas** [open-olive/loop-development-kit#482]
Hi! I am currently trying to make the text of a link bold. This does not currently seem to be possible with the link component.
I know that with a markdown component, if you have a valid url, you can get a bold purple link. But the problem is that my link isn't an actual url. It's just a link that causes some redux actions to be dispatched (see code below).

```
<oh-link
  key="provider-name-link"
  text={`${provider.getNameWithTypes()}`}
  onClick={() => {
    // set selectedProvider on provider directory state
    dispatch(setSelectedProvider(provider));
    dispatch(updateProviderLocationPageNumber(0));
    dispatch(updatePage(Page.PROVIDER_DETAIL));
  }}
/>
```

Is it possible to add an optional property to the link component to allow for the text to be bold? Thanks!

**Comment by JosefBud-olive** [comment]
Hey there, @kyuhas! This is coincidentally a feature that's in active development right now actually! I'll tag the ticket with this issue to let you know here when it's complete.

**Comment by kyuhas** [comment]
@JosefBud-olive oh that's amazing! Thank you for the update!

**Comment by MaggieMaSS** [comment]
Thanks for your feedback! The ticket number is HELPS-2848. We have released this feature on Olive Helps 0.31.2 and LDK v3.11.0 :)
